src/main/resources/application.properties

```diff
@@ -1,10 +1,7 @@
-spring.h2.console.enabled=true
 spring.jpa.hibernate.ddl-auto=update
 spring.jpa.properties.hibernate.format_sql=true
 logging.level.org.hibernate.SQL=DEBUG
 logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE
-spring.datasource.url=jdbc:mysql://localhost:3306/testdb?useSSL=false&serverTimezone=UTC
+spring.datasource.url=jdbc:mysql://localhost:3306/switchboard?useSSL=false&serverTimezone=UTC
```

Comment:
I feel like it would be ideal if the database name (and password) were easily configurable locally. anyone have ideas on how to do that? maybe a third `application.properties` that's gitignored so we can override there? or just IDE runtime configuration overrides?

Reply:
Yes, we should all have some `.properties` file that is gitignored which includes credentials.

Reply:
Actually, on second thought, we can simply use the same `applications.properties` with TODO comment <INSERT CREDENTIALS> placeholder for the username and password property values

Reply:
@felixlapierre see last commit

Reply:
We could explore better options down the road, but for now I think this works

Reply:
The problem with using TODO is that when we fill in the value locally it will always appear as a change that is ready to be committed. So you have to be careful not to accidentally commit your local username & password which is very annoying.

Reply:
IMO we can leave it as default values for now, which will work for most people's setups, and come up with a better solution in a later PR.

Reply:
@felixlapierre done